README.md

```diff
@@ -13,3 +13,17 @@ Planned features:
 - parse IdP-configuration from file/url (manually via CLI or on configurable timer)
 - parsed IdP-configurations persist in database
 - activate/deactivate some IdPs at your choice (e.g. if your security-team would like to allow IdPs only after taking a look at them)
+
+## Installation
+
+Use your package installer to install `invenio_edugain`, e.g. via `pip`:
+
+```bash
+pip install invenio_edugain
+```
+
+When adding to an existing instance, you'll have to create the new SQL-tables used by `invenio_edugain`:
+
+```bash
+invenio db create
```

Comment:
maybe better to say to use `invenio alembic upgrade`?

Reply:
Agree that that's better.
Added initial alembic revisions for this and updated README.